data/mods/Magiclysm/monsters/Ogre.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "id": "mon_ogre",
+    "type": "MONSTER",
+    "name": { "str": "Ogre" },
+    "description": "Green beast known for living in swamps and eating humans, it will kill anything it gets its hands on.",
+    "default_faction": "ogres",
+    "bodytype": "human",
+    "species": [ "OGRE" ],
+    "volume": "100 L",
+    "weight": "130 kg",
+    "hp": 140,
+    "speed": 123,
+    "material": [ "flesh" ],
+    "symbol": "O",
+    "color": "green",
+    "aggression": 123,
+    "morale": 113,
+    "melee_skill": 7,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
```

Comment:
I think this is going to do very little to no damage and going to be impossible to hit without guns because I think 2 3 sided dice is a low damage dealing quality and dodge 20 is twice the dodge maximum I think.